<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MacroEditorUI</class>
 <widget class="QWidget" name="MacroEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macro Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MacroApp.qrc">
    <normaloff>:/MacroApp/logo.png</normaloff>:/MacroApp/logo.png</iconset>
  </property>
  <widget class="QLabel" name="macroNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Macro Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="interceptedKeyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Intercepted Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="enabledLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="enabledBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="actionLabel">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Actions</string>
   </property>
  </widget>
  <widget class="QListWidget" name="actionList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newActionButton">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>260</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>New Action</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteActionButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Action</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveNameButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="interceptedKeyText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseKeyButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="macroNameText">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MacroApp.qrc"/>
 </resources>
 <connections/>
</ui>
